db/migrate/20230310151243_create_deletion_request.rb

```diff
@@ -0,0 +1,12 @@
+class CreateDeletionRequest < ActiveRecord::Migration[6.1]
+  def change
+    create_table :deletion_requests do |t|
+      t.string :provider
+      t.string :uid
+      t.string :pid
```

Comment:
What does that stand for, "provider ID"?

Reply:
public id

Reply:
it's the code that can be used to query to check the status